_clients/data-prepper/get-started.md

```diff
@@ -9,15 +9,42 @@ nav_order: 1
 
 Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
 
-## 1. Install Data Prepper
+If you are migrating from Open Distro Data Prepper, visit the [Migrating from Open Distro](migrating_from_opendistro.md) page. Otherwise, please continue.
 
-To use the Docker image, pull it like any other image:
+## 1. Installing Data Prepper
 
-```bash
+There are two ways to install Data Prepper for running:
+
+1. Run the Docker image
+2. Build from source
+
+The easiest way to use Data Prepper is by running the Docker image. We suggest
+you use this approach if you have [Docker](https://www.docker.com) available.
+
+You can pull the Docker image:
+
+```
 docker pull opensearchproject/data-prepper:latest
 ```
 
-## 2. Define a pipeline
+If you have special requirements that require you build from source, or if you
+want to contribute, please see the [Developer Guide](developer_guide.md).
```

Comment:
This Developer Guide is located in the Data Prepper GitHub repository. So we should link to it using a fully-qualified URL:

https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md